packages/ob-sync/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@badget/tsconfig/base.json",
```

Comment:
`@projectx/tsconfig/base.json` type

Reply:
Oh, sorry Bad. I missed it. Thanks for catching it